targets/xiwi

```diff
@@ -28,8 +28,27 @@ if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
     apt-mark unhold xserver-xorg-video-dummy$ltspackages || true 2>/dev/null
```

Comment:
No, but it might make more sense to add the patches as a user patch so that every time it's updated, it'll just automatically apply. I'll look into doing this instead for this package.